core/base/distanceMatrixDistorsion/DistanceMatrixDistorsion.h

```diff
@@ -33,10 +33,10 @@ namespace ttk {
   public:
     DistanceMatrixDistorsion();
 
-    int execute(const std::vector<std::vector<double>> &highDistMatrix,
-                const std::vector<std::vector<double>> &lowDistMatrix,
+    int execute(const std::vector<double *> &highDistMatrix,
```

Comment:
should the `double` type made more generic (i.e. `THigh` and `TLow`, similarly to LDistanceMatrix)

Reply:
Small question: is the overhead of templating (compilation time, maybe executable size) worth it?
Same as above, if you believe it, I'll do the changes.

Reply:
at the moment ttkLDistanceMatrix and others (ttkPersistenceDiagramDistanceMatrix, ttkMergeTreeDistanceMatrix) all see to produce distance matrices encoded with double. so ok for keeping doubles then.